api/src/test/java/io/kaoto/backend/api/resource/v2/IntegrationsResourceTest.java

```diff
@@ -0,0 +1,393 @@
+package io.kaoto.backend.api.resource.v2;
```

Comment:
I copied the `v1` file here thinking at some point we will deprecate and remove `v1` and its tests.

At the end the round trip function.